### ISSUE TITLE
Blur Featured Image when it doesn't have alt text

### ISSUE THREAD

**Issue by mrwweb** [kerri-hicks/blur_no-alt#8]
Title pretty much says it all. I haven't looked into how to do this, but I assume the image in the markup contains alt text.

**Comment by mrwweb** [comment]
I briefly looked into this just now. It appears that the Featured Image setting in the sidebar has empty alt text regardless of whether the image has alt text in the Media Library or not. Further, since the Featured Image output is so theme dependent, it seems like we can't really know whether a Featured Image _should_ have alt text. And even if it does, we can't know if the theme actually displays it. So taking that all into account, I don't think this is a good idea. I'm going to close this issue.
